tests/acceptance/21_methods/action_ifelapsed_respected.cf.sub

```diff
@@ -0,0 +1,23 @@
+bundle agent main
+{
+  methods:
+    "test"
+      action => expireafter_ten_sec;
+}
+
+body action expireafter_ten_sec
+{
+  expireafter => "10";
```

Comment:
@nickanderson I agree with @eysteins here, you need to set this to "1" minute and sleep something like 120 seconds.